pkg/xen-tools/Dockerfile.in

```diff
@@ -14,18 +15,19 @@ RUN gcc -s -o /chroot2 /tmp/chroot2.c
 RUN gcc -s -o /hacf /tmp/hacf.c
 RUN mkinitfs -n -F base -i /init-initrd -o /runx-initrd
 
-FROM alpine:3.12 as kernel-build
-ENV GIT_HTTP=y
-# hadolint ignore=DL3018
-RUN apk add --no-cache \
+FROM lfedge/eve-alpine:ce776885c43588292126c2fcc7300b77a494faf3 as build
```

Comment:
Why do you use different tag of eve-alpine here and few lines above (not 99db...)?
More general question: can we use something like ALPINE_TAG in such places?

Reply:
The real answer is that once it is settled we're going to use `lfedge/eve-alpine:6.x.0` and we will leave it like that for a loooooong time (hopefully) @petr-zededa